app/pages/foreign/ForeignPropertySummaryPage.scala

```diff
@@ -16,25 +16,40 @@
 
 package pages.foreign
 
-import models.NormalMode
+import models.{NormalMode, UserAnswers}
 import viewmodels.summary.{TaskListItem, TaskListTag}
 
-case class ForeignPropertySummaryPage(taxYear: Int, startItems: Seq[TaskListItem], foreignIncomeCountries: List[Country])
-
-
+case class ForeignPropertySummaryPage(
+  taxYear: Int,
+  startItems: Seq[TaskListItem],
+  foreignIncomeCountries: List[Country]
+)
 
 object ForeignPropertySummaryPage {
 
-  def propertyAboutItems(taxYear: Int): Seq[TaskListItem] =
+  def foreignPropertyAboutItems(taxYear: Int, userAnswers: Option[UserAnswers]): Seq[TaskListItem] = {
```

Comment:
Hey Vivek could you please add some unit tests for this

Reply:
Added @tapiwa-tiyemba